src/Lykke.Job.BlockchainCashinDetector.Core/Domain/CashinAggregate.cs

```diff
@@ -192,12 +192,14 @@ public bool Start(decimal balanceAmount, long balanceBlock, decimal enrolledBala
 
             if(!couldBeStarted)
             {
-                return false;
+                Result = CashinResult.OutdatedBalance;
```

Comment:
Is it correct to assign `Result` without switching a state and switch state without assigning a result?

Reply:
It's not must have here, but for traceability it will be nice to add extra state. Will do in separate task